restapi/src/main/java/io/stargate/web/docsapi/exception/ErrorCode.java

```diff
@@ -35,6 +35,9 @@
       "The table name contains invalid characters. Valid characters are alphanumeric and underscores."),
 
   /** Document API. */
+  DOCS_API_TABLE_NOT_A_COLLECTION(
+      Response.Status.BAD_REQUEST,
+      "The requested Cassandra table is not a Documents API Collection."),
```

Comment:
Minor but this should be more generic
```suggestion
      "The requested database table is not a Documents API Collection."),
```